packages/ui/commerce/hooks/useUserPrice.tsx

```diff
@@ -32,15 +41,17 @@ export function useGetUserPrice() {
             discountTier != null && isProUser
                ? proPricesByTier?.[discountTier]
                : null;
-         if (proPrice == null) {
+         if (proPrice == null || price.amount < proPrice.amount) {
```

Comment:
I don't think this is right..  i.e. I don't think they will *get* the pro price during checkout even if it's higher. I'd imagine they would get the price from their discount tier? I don't know what controls which price they get in shopify, but it feels likely unintentional. Do we set lower prices for normal users (a sale) and keep the pro price high occasionally? Is that inentional? @jeffsnyder

Reply:
I guess the short of my comment is:

**We should show them the price that they are going to receive when they check out**. I don't know the specifics of what that price is, but I doubt it's `min(all the prices)`.

Reply:
> We should show them the price that they are going to receive when they check out.

Good thought, I'll have QA check this

Reply:
I feel a little embarrassed that I don't already know, but I feel like we ought to look at the code / docs / ... and figure out what our code is trying to do (i.e. how folks are charged different prices).